src/applications/discharge-wizard/components/v2/questions/shared/Dropdown.jsx

```diff
@@ -74,6 +77,9 @@ const Dropdown = ({
 
         if (forkableQuestions.includes(shortName) && editMode) {
           toggleQuestionsFlowChanged(true);
+          toggleAnswerChanged(true);
```

Comment:
Consider: I've noticed in a few files in this PR that we're duplicating `toggleAnswerChanged(true)` on both sides of an `if` statement. Is this for readability / clarity? Perhaps we could consolidate and place it once above the `if` statement.

Reply:
Thanks! I'll clean this up it needs to be in the if statement only. But I can rewrite it to only write it once.